### PR TITLE
Small code cleanup

### PR DESCRIPTION
**Pull request by isinyaaa**
This is a small cleanup I made during PR's #543 development.

### CODE REVIEW

File: src/config_manager.sh [hunk 1]
```diff
@@ -296,7 +296,6 @@ function get_config_from_defconfig()
   # Build command
   [[ -n "$arch" ]] && cmd+=" ARCH=$arch"
   [[ -n "$cross_compile" ]] && cmd+=" CROSS_COMPILE=$cross_compile"
-  [[ -n "$asic" ]] && cmd+=" $asic\_defconfig"
```
Comment (kwy95) [inline src/config_manager.sh:299]: Shouldn't this PR also remove line 294 from this file?

**Review by kwy95 — COMMENTED** [review]
I think there is an extra line that should be removed based on the comments from the previous PR, apart from that this LGTM.

**Comment by isinyaaa** [comment]
@kwy95 thanks for the quick review :) I just think I might need to update some other stuff as I'm rebasing this on some months of other contributors' work, and there might be other stuff that won't work out of the box.

**Comment by rodrigosiqueira** [comment]
Hi @isinyaaa , thanks a lot for this PR. I made some minor changes in the commit number 3 and merged it into the unstable branch.

**Pull request closed without merging**